src/renderer/qssWidget/js/qssMouseWidget.js

```diff
@@ -0,0 +1,223 @@
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            swapMouseButtons: {
+                type: "gpii.qssWidget.mouseWidgetToggle",
+                container: "{that}.dom.swapMouseButtons",
+                options: {
+                    model: {
+                        setting: "{mouse}.model.setting.settings.swapMouseButtons"
+                    }
+                }
+            },
+            easierDoubleClick: {
+                type: "gpii.qssWidget.mouseWidgetToggle",
+                container: "{that}.dom.easierDoubleClick",
+                options: {
+                    model: {
+                        setting: "{mouse}.model.setting.settings.easierDoubleClick"
+                    }
+                }
+            },
+            largerMousePointer: {
+                type: "gpii.qssWidget.mouseWidgetToggle",
+                container: "{that}.dom.largerMousePointer",
+                options: {
+                    model: {
+                        setting: "{mouse}.model.setting.settings.largerMousePointer"
+                    }
+                }
+            },
+            channelNotifier: {
+                type: "gpii.psp.channelNotifier",
+                options: {
+                    events: {
+                        // Add events the main process to be notified for
+                        onQssWidgetSettingAltered:       "{mouse}.events.onQssWidgetSettingAltered",
+                        onQssWidgetNotificationRequired: "{mouse}.events.onQssWidgetNotificationRequired"
+                    }
+                }
+            }
+        }
+    });
+
+    /**
+     * Represents the QSS mouse toggle widget.
+     */
+    fluid.defaults("gpii.qssWidget.mouseWidgetToggle", {
+        gradeNames: ["fluid.viewComponent", "gpii.psp.selectorsTextRenderer"],
+
+        selectors: {
+            toggleButton: ".flc-toggleButton",
+            settingTitle: ".flc-qssMouseWidgetToggle-settingTitle"
+        },
+
+        enableRichText: true,
+
+        model: {
+            setting: {},
+            value: "{that}.model.setting.value",
+            messages: {
+                settingTitle: "{that}.model.setting.schema.title"
+            }
+        },
+
+        components: {
+            toggleButton: {
+                type: "gpii.psp.widgets.switch",
+                container: "{that}.dom.toggleButton",
+                options: {
+                    model: {},
+                    modelRelay: {
+                        "enabled": {
+                            target: "enabled",
+                            singleTransform: {
+                                type: "fluid.transforms.valueMapper",
+                                defaultInput: "{mouseWidgetToggle}.model.value",
+                                match: {
+                                    500: false,
```

Comment:
Strange to map these literal values 500, 5000 to false and true, I am guessing these correspond to some values from the schema in which case they should be referenced from there directly